impeller/aiks/canvas.cc

```diff
@@ -54,6 +57,14 @@ void Canvas::Initialize() {
   FML_DCHECK(base_pass_->GetSubpassesDepth() == 1u);
 }
 
+void Canvas::SetEnableOffscreenCheckerboard(bool enabled) {
```

Comment:
We could have this and other debugging options separated into a debug options struct that can be assigned to the canvas.

Reply:
Done.